package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "install": "yarn run build",
     "config-xcode": "node-gyp configure -- -f xcode",
-    "build": "node-gyp rebuild --target=4.0.1 --arch=x64 --dist-url=https://atom.io/download/electron --openssl_fips=X && genversion --es6 --semi version.generated.ts",
+    "build": "node-gyp rebuild --target=4.0.1 --arch=x64 --dist-url=https://electronjs.org/headers --openssl_fips=X && genversion --es6 --semi version.generated.ts",
```

Comment:
The changed url returns the following, I don't think it should return that...

```xml
<Error>
  <Code>BlobNotFound</Code>
  <Message>The specified blob does not exist. RequestId:05f7ba07-901e-008d-48ad-29e8bd000000 Time:2023-01-16T13:22:47.4260054Z</Message>
</Error>
```

Reading the blog post, shouldn't it point to something like `https://artifacts.electronjs.org/headers`?